common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -328,11 +339,12 @@ func (m *cassandraMetadataPersistenceV2) GetDomain(request *p.GetDomainRequest)
 	if info.Data == nil {
 		info.Data = map[string]string{}
 	}
+	config.BadBinaries = p.NewDataBlob(badBinariesData, common.EncodingType(badBinariesDataEncoding))
```

Comment:
do you need  "!= nil" check or can the DataBlob can take nil ?

Reply:
NewDataBlob inside already handled that : https://github.com/uber/cadence/blob/master/common/persistence/persistenceInterface.go#L589